stl/inc/set

```diff
@@ -49,114 +47,111 @@ public:
     }
 };
 
-// CLASS TEMPLATE set
 template <class _Kty, class _Pr = less<_Kty>, class _Alloc = allocator<_Kty>>
-class set : public _Tree<_Tset_traits<_Kty, _Pr, _Alloc, false>> {
-    // ordered red-black tree of key values, unique keys
-public:
-    static_assert(!_ENFORCE_MATCHING_ALLOCATORS || is_same_v<_Kty, typename _Alloc::value_type>,
-        _MISMATCHED_ALLOCATOR_MESSAGE("set<T, Compare, Allocator>", "T"));
-
-    using _Mybase                = _Tree<_Tset_traits<_Kty, _Pr, _Alloc, false>>;
-    using key_type               = _Kty;
-    using key_compare            = _Pr;
-    using value_compare          = typename _Mybase::value_compare;
-    using value_type             = typename _Mybase::value_type;
-    using allocator_type         = typename _Mybase::allocator_type;
-    using size_type              = typename _Mybase::size_type;
-    using difference_type        = typename _Mybase::difference_type;
-    using pointer                = typename _Mybase::pointer;
-    using const_pointer          = typename _Mybase::const_pointer;
-    using reference              = value_type&;
-    using const_reference        = const value_type&;
-    using iterator               = typename _Mybase::iterator;
-    using const_iterator         = typename _Mybase::const_iterator;
-    using reverse_iterator       = typename _Mybase::reverse_iterator;
-    using const_reverse_iterator = typename _Mybase::const_reverse_iterator;
-
-    using _Alnode        = typename _Mybase::_Alnode;
-    using _Alnode_traits = typename _Mybase::_Alnode_traits;
+// ordered red-black tree of key values, unique keys
+public : static_assert(!_ENFORCE_MATCHING_ALLOCATORS || is_same_v<_Kty, typename _Alloc::value_type>,

```

Comment:
This does not make sense

Reply:
I think VS Code might not have liked replacing so many....

(to note, I did audit the previews for each file before replacing)

Reply:
it is still not resolved